### DISCUSSION TITLE
Commande SQL via JAVA

### DISCUSSION THREAD

**Discussion by lucile98-ode** [lucile98-ode/GPI-L2BHA-FENOUIL#27] [Q&A] [answered]
Pour les commandes SQL (SELECT, ...) via le langage JAVA est assez complexe par l'implémentation de plusieurs classes.
@chrabou 

Utilisation de documentation ainsi que de videos explicatives afin de parvenir à l'objectif.

**Comment by lucile98-ode** [comment, marked as answer]
suggestion : mise en place de plusieurs méthode dans la classe de liaison pour répondre à chaque demande. @chrabou @HaceneM

**Reply by lucile98-ode** [reply]
appelle de ses méthodes dans les classes correspondantes @amrx12

**Comment by chrabou** [comment]
Les vidéos en anglais sont mieux explicatives vis a vis du MySQL pour faire les commandes pour le code JAVA.
